domains/vortex.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "elgyro",
+           "email": "",
+           "discord": "741259388999696484",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.FlieH5n7ONBRl7dBlHKiNXUYGJUNp2prTbBinF0tWYexWC9RJrPQQrZLvIM9IBCElSgl1p0nlWS6BFLgIzASV8oy6FHOjkYE1TWZpHHCMlMV8el8oqGn6AKOqztky699Tj_eNe4FaoJYkR5YiSvDZ7lvPkjWfFLughjgseYjr99KxFx2TZUgwzkjgc3fFDefI6KiVnZjNeX8a7WoXtCpBc_BEBnHcK4_yVYOYoJOo_EL_65CZ_MZdizlYQpQfGoimEnXsFnk-DIowahiVaiIpl3PxeUG8YMfB-lOgxtft6NS6lBffo4zKfBvOVfHKNca-Q2llT1QzcNQBc1SKHvLpQ.6MoF80ffbdJSv-qUHUA10g.QJcnxnVO58jKhhjycHKn08cRUmedVHXUPCnK95svgAqZbDrwyD5oJKT5vz6-9MPT1sUp--SdxveUziWW-l6p79yJfKlnJ_SlVjEV_4FKchw.eSiz5AbbI_WHnR6pQ3qPgw"
+        },
+    
+        "record": {
+            "A": ["185.199.108.153"]
```

Comment:
I think there should be 4 Github Pages IP Address.
Also you need to give a preview of your website.